nixos/modules/services/security/fail2ban.nix

```
@@ -137,8 +143,8 @@ in
       };
 
       bantime-increment.factor = mkOption {
-        default = "1";
-        type = types.str;
+        default = 1;
+        type = types.int;
         example = "4";
```

Keeping backwards compatibility here is straightforward enough:

```suggestion
        type = with types; coercedTo str builtins.fromJSON int;
        example = 4;
```

---

I left it as a string for more simplicity